slurm_ops_manager/slurm_snap_manager.py

```diff
@@ -123,7 +123,7 @@ def upgrade(self):
         # instead).
         self.setup_system()
 
-    def _provision_snap_systemd_service_override_file(self):
+    def write_systemd(self):
```

Comment:
I would like to see a more descriptive function name here.... I actually liked your first go, how about `write_systemd_service_override_file()`?

Reply:
this function should go in slurm_ops_base and use `self._slurm_systemd_service` to template the override.conf location.

Reply:
add a doc string to this function

Reply:
I added a doc string to make function more descriptive